src/main/fc/fc_msp.c

```diff
@@ -287,7 +288,12 @@ static void serializeDataflashSummaryReply(sbuf_t *dst)
 }
 
 #ifdef USE_FLASHFS
-static void serializeDataflashReadReply(sbuf_t *dst, uint32_t address, const uint16_t size, bool useLegacyFormat)
+enum compressionType_e {
+    NO_COMPRESSION,
+    HUFFMAN
+};
+
+static void __attribute__((optimize("-O0"))) serializeDataflashReadReply(sbuf_t *dst, uint32_t address, const uint16_t size, bool useLegacyFormat)
```

Comment:
Previously I had no issues with it, besides I don't see how LTO could affect local variables in functions with exactly one caller.
Something has changed after @blckmn's work on Makefiles or around this point.